interview-questions.md

```diff
@@ -8,48 +8,48 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is JSX? What is one syntax difference between HTML and JSX?
 
-Your answer:
+Your answer: JSX is javascript in react. {} is required to write javascript inside of an html area
 
-Researched answer:
+Researched answer: JSX is what allows React to create user interfaces. JSX uses className while HTML uses class. this is because class already means something else inside javascript.
 
 2. What is yarn? What file(s) are modified in a React application when you run the command yarn in your terminal?
 
-Your answer:
+Your answer: yarn is an application that is reached thru terminal. you get yarn.lock.
 
-Researched answer:
+Researched answer: A javascript testing framework. the files test.js node_modules, package.json and yarn.lock
 
 3. What is an anonymous function in JavaScript?
 
-Your answer:
+Your answer: () => {}
 
-Researched answer:
+Researched answer: a function with out any name associated with it.
 
 4. What is the difference between state values and props in React?
 
-Your answer:
+Your answer: state values hold information given to it and a prop is information that is passed up the river of information.
 
-Researched answer:
+Researched answer: props are passed up to the main file from other components. state is used on the main file.
 
 5. What is the DOM?
 
-Your answer:
+Your answer: the thing that records any user events on the page.
```

Comment:
Currently, this section is completing the bare minimum requirements for this section.  This section is meant to be a reflection on how you would answer in an interview, and one line answers will often not impress an interviewer.